apps/omg_watcher/lib/exit_processor/in_flight_exit_info.ex

```diff
@@ -180,6 +177,28 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
     {ife_hash, struct!(__MODULE__, ife_map)}
   end
 
+  # NOTE: the databases currently don't hold the `signed_tx_bytes` field, hence dropping this here and in the other fun.
+  # NOTE: non-private because `CompetitorInfo` holds `Transaction.Signed` objects too
+  def from_db_signed_tx(%{raw_tx: raw_tx_map, sigs: sigs}) when is_list(sigs) do
```

Comment:
there is only one name for `to_db_value` methods and pattern matching chooses the correct one, but methods `from_db_*_tx` have different names. Could you choose one approach?

Reply:
in fact, I can't, because `from_*` would need to know what type it reads, while it is always a map. So the caller needs to decide, what it expects. OTOH calling it as `from_db_value(Transaction.Signed, map)` would go against the other convention, where these are always single argument functions. So this way or another, we're breaking some conventions